packages/terra-application/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.52.2",
   "description": "A framework to support application development with Terra components",
   "engines": {
-    "node": ">=8.9.2 <15"
+    "node": "^8.9.2 || ^10.13.0 || ^12.0.0 || ^14.0.0"
```

Comment:
Four versions seems a bit much. We are supporting three versions in other places - why not make it easy on ourselves and drop 8.0. Node 8 is [circa 2017](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V8.md#8.0.0).

Reply:
We actually got rid of Node 8 when we upgraded this to Node 12. See change [here](https://github.com/cerner/terra-application/pull/225/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R7). We later realized we broke our consumers because some are still on Node 8 so we added Node 8 back on this [PR #227](https://github.com/cerner/terra-application/pull/227/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R7). So to continue to remain passive, we should continue to keep Node 8. We can remove it in our next MVB.

Reply:
Thanks. This motivates me to continue crafting [a deprecation policy](https://github.com/cerner/terra-ui/issues/413).